static/src/js/components/DownloadHistory/DownloadHistory.jsx

```diff
@@ -16,144 +16,131 @@ import PortalLinkContainer from '../../containers/PortalLinkContainer/PortalLink
+          <Spinner
+            dataTestId="spinner"
+            className="download-history__spinner"
+            type="dots"
+            size="small"
+          />
+        )
       }
-
-      // If more than one collection exists show the title of the first
-      return `${title} and ${collections.length - 1} other ${pluralize('collection', collections.length - 1)}`
-    }
-
-    // No collection title exists so we'll just display how many collections were downloaded
-    return `${collections.length} ${pluralize('collection', collections.length)}`
-  }
-
-  render() {
-    const {
-      earthdataEnvironment,
-      retrievalHistory,
-      retrievalHistoryLoading,
-      retrievalHistoryLoaded
-    } = this.props
-
-    const { edscHost } = this
-
-    return (
-      <>
-        <Helmet>
-          <title>Download Status &amp; History</title>
-          <meta name="title" content="Download Status &amp; History" />
-          <meta name="robots" content="noindex, nofollow" />
-          <link rel="canonical" href={`${edscHost}/downloads`} />
-        </Helmet>
-        <h2 className="route-wrapper__page-heading">Download Status & History</h2>
-        {
-          (retrievalHistoryLoading && !retrievalHistoryLoaded) && (
-            <Spinner
-              className="download-history__spinner"
-              type="dots"
-              size="small"
-            />
-          )
-        }
-        {
-          retrievalHistoryLoaded && (
-            retrievalHistory.length > 0 ? (
-              <Table className="download-history-table">
-                <thead>
-                  <tr>
-                    <th className="download-history-table__contents-heading">Contents</th>
-                    <th className="download-history-table__created-heading">Created</th>
-                    <th className="download-history-table__actions-heading">Actions</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  {
-                    retrievalHistory.map((retrieval) => {
-                      const {
-                        id,
-                        created_at: createdAt,
-                        jsondata,
-                        collections
-                      } = retrieval
-
-                      const { portal_id: portalId } = jsondata
-
-                      return (
-                        <tr key={id}>
-                          <td>
-                            <PortalLinkContainer
-                              portalId={portalId}
-                              to={
-                                {
-                                  pathname: `/downloads/${id}`,
-                                  search: stringify({ ee: earthdataEnvironment === deployedEnvironment() ? '' : earthdataEnvironment })
-                                }
+      {
+        retrievalHistoryLoaded && (
+          retrievalHistory.length > 0 ? (
```

Comment:
I think a turnary makes this hard to follow. Something more explicit like this might work better:

```
{
retrievalHistoryLoaded && retrievalHistory.length > 0 && (
...
)
retrievalHistoryLoaded && retrievalHistory.length === 0 && (
...
)
}
```